public/js/app.js

```diff
@@ -32,7 +32,7 @@ $(function(){
   // });
 
   $('.sequence-expand-button').click(function (event) {
-    event.preventDefault();
+    event.preventDefault ? event.preventDefault() : (event.returnValue = false);
```

Comment:
Should we do this instead?  https://github.com/siteadmin/inferno/compare/development...a-to-button-example

I didn't merge because of conflicts.  If you agree, just grab that code and add it in here?  Anchor tags shouldn't be used for things that aren't external links.